Ayo/loginpage/style.css

```
@@ -9,52 +9,92 @@ a, a:hover, a:focus a:visited {
   color: inherit;
 }
 
+button:focus {
+  outline: none;
+}
+
 body {
-  margin:0px;
-  padding:0px;
+  margin: 0;
+}
+
+.bg-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 1;
   background: url("background.jpg") no-repeat;
   background-size: cover;
+  -webkit-filter: blur(5px);
+  -moz-filter: blur(5px);
+  -o-filter: blur(5px);
+  -ms-filter: blur(5px);
+  filter: blur(5px);
 }
 
 .container {
+  top: 56px;
+  left: 0;
+  right: 0;
+  z-index: 10;
   background-color: white;
   width: 420px;
   height: 594px;
   position: relative;
-  margin: auto;
-  margin-top: 56px;
+  margin: 0 auto;
   text-align: center;
-  display: flex;
-  flex-flow: column wrap;
+  padding-bottom: 16px;
+  padding-left: 85px;
+  padding-right: 85px;
+  padding-top: 56px;
+}
+
+.top {
+  margin-bottom: 93px;
 }
 
 .mainPic {
-  margin-top: 56px;
+  margin: 0 auto;
+  text-align: center;
   margin-bottom: 34px;
+  height: 65px;
+}
+
+img {
+  height: 65px;
 }
 
 .topic {
   margin: 0 auto;
-  margin-bottom: 93px;
   text-align: center;
-  font-size: 16px;
   letter-spacing: 1.2px;
-  color: rgba(0, 0, 0, 0.87);　
+  color: rgba(0, 0, 0, 0.87);
+}
+
+h1 {
+  font-size: 16px;
+  font-weight: 1;
+  margin: 0 auto;
 }
 
 .formGroup {
   display: flex;
   flex-flow: column wrap;
```

flex-flow 是 column，畫面上也只是單純的從上到下排版，就不需要 wrap
```suggestion
  flex-flow: column;
```